### ISSUE TITLE
how about the code of the model?

### ISSUE THREAD

**Issue by M-leng** [eldentse/collab-hand-object#2]

**Comment by M-leng** [comment]
I have sent an email with the author of the paper. I listed some question, he is very nice to give me detail answers. Our discussion is shown here:

Question:
1) With the attention graph convolution, if the shape of hand mesh features is b*778*64, how to change it to b*64, so that it can be concatenated with image features B*512. By max pooling or mean pooling?

2) When the network, for the object branch, what is the loss? Like Hasson, including chamfer distance, edge loss and laplacian loss? If you use edge loss and laplacian loss, what is the weight for them? And do you use the contact loss as Hasson?

3) I have read the code of Hasson, there are so many manual setting parameters for training, like "/mano_train/options/net3dopts.py". Could you tell me your parameters for this file?

Answer:


    1.I remembered that I did not use any pooling operations. I have used FC to resize the correct size for addition.
    2.Loss functions that I have used are all in the paper. The object related losses are adjusted according to the dimension of the dataset. I will need to double check my machine to find the exact weights. I did not use contact loss from Hasson because I wanted to train on also pre-grasp stage of dataset.
    3. so please give me need some time to obtain the files for you.
    4. Regarding to the picture, I have experienced something similar too. There are multiple reasons. But first please check implementation details and especially associative loss. Because the network can find very difficult to converge without it.
